src/devices/Common/Iot/Device/Graphics/LcdCharacterEncoding.cs

```diff
@@ -102,7 +102,7 @@ public override int GetBytes(char[] chars, int charIndex, int charCount, byte[]
                 }
                 else
                 {
-                    bytes[byteIndex] = _characterMapping.GetValueOrDefault(_unknownLetter);
+                    bytes[byteIndex] = _characterMapping[_unknownLetter];
```

Comment:
Should we add a check in case this index doesn't exist? that is what GetValueOrDefault used to do

Reply:
Yes. There is already a check in the factory that typically generates this class, but since the ctor is public, I added a validation there. The "unknown" letter must exist.